base/commands/migration/migration_stages.go

```diff
@@ -76,6 +77,13 @@ func createMigrationStages(ctx context.Context, ec plug.ExecContext, ci *hazelca
 					case StatusCanceled, StatusCanceling:
 						execErr = clcerrors.ErrUserCancelled
 						break statusReaderLoop
+					case StatusInProgress:
+						rt, cp, err := fetchOverallProgress(ctx, ci, migrationID)
+						if err != nil {
+							ec.Logger().Error(err)
+						}
+						status.SetProgress(cp)
+						status.SetRemainingDuration(rt)
```

Comment:
If there's an error the progress and the remaining duration would be set to 0.
Is that the expected outcome?

Reply:
Shall we set the text if there is an error and not set progress and duration:
```
status.SetText("Unable to calculate remaining duration and progress")
```

Reply:
If we can't fetch the progress, I would not update them.

Reply:
Yeah, maybe something like that.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/423/commits/9a153d5ad95ee2477f02f35194241ad60932afb4